src/shock95x/auctionhouse/menu/type/AHMenu.php

```diff
@@ -81,17 +81,8 @@ protected function openListing(int $slot, Item $itemClicked): bool {
 			Await::f2c(function () use ($itemClicked, $slot) {
 				$plugin = AuctionHouse::getInstance();
 				$listing = $this->getListings()[$slot];
-				$balance = yield $plugin->getEconomyProvider()->getMoney($this->player, yield) => Await::ONCE;
```

Comment:
imo this check is unreasonable, because even though player has insufficient money before confirmation, they may have received it just during the confirmation dialog.